flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/EventRecorder.java

```diff
@@ -280,7 +280,8 @@ public KubernetesClient getKubernetesClient() {
     /** The type of the events. */
     public enum Type {
         Normal,
-        Warning
+        Warning,
+        Error
```

Comment:
We could keep using the `Warning` type here. I wouldn't consider these events more severe than other operator errors that are recorded as warnings especially that the job often recovers from them